x-pack/test/api_integration/apis/infra/logs_without_millis.ts

```diff
@@ -41,66 +33,25 @@ const LATEST_KEY_WITH_DATA = {
 
 export default function({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
-  const client = getService('infraOpsGraphQLClient');
   const supertest = getService('supertest');
 
   describe('logs without epoch_millis format', () => {
     before(() => esArchiver.load('infra/logs_without_epoch_millis'));
     after(() => esArchiver.unload('infra/logs_without_epoch_millis'));
 
-    it('logEntriesAround should return log entries', async () => {
```

Comment:
We don't have that endpoint anymore 🙂

Reply:
Because the functionality was folded into `/api/log_entries/entries`? Then that regression probably applies there as well, because the underlying query barely changed.

Reply:
I see.

I made a meta issue where I'll follow up on all of these, unless you think it's critical and needs to be done now.

Reply:
Nah, I brought the test back in this PR.